scripts/bug_retriever.py

```diff
@@ -60,15 +60,22 @@ def retrieve_bugs(self):
 
         # Try to re-download inconsistent bugs, up to three times.
         for i in range(3):
-            bug_ids = set(bug_snapshot.get_inconsistencies())
-            if len(bug_ids) == 0:
+            # We look for inconsistencies in all bugs first, then, on following passes,
+            # we only look for inconsistencies in bugs that were found to be inconsistent in the first pass
+            if i == 0:
+                inconsistent_bugs = bugzilla.get_bugs()
+            else:
+                inconsistent_bugs = inconsistent_bugs
```

Comment:
You can definitely remove this line, assigning a variable to itself doesn't have any effect :P

Reply:
Yeah , But again our main agenda won't work that in second loop we should pass parameters which are inconsistent at first time.

Reply:
I'm not sure what you mean.

Reply:
Sorry, I got your point.